cypress/apiRequests/Product.js

```diff
@@ -17,44 +17,50 @@ class Product {
                 }
               }
             }
-          }
-        `;
+          `;
     return cy
       .sendRequestWithQuery(query)
       .then(resp => resp.body.data.products.edges);
   }
 
-  updateChannelInProduct(productId, channelId) {
+  updateChannelInProduct(
```

Comment:
In case we add more than 3 args, I'd go for passing them as an object - that way it's much easier to avoid passing them in wrong order, especially without typing. And then do:
```suggestion
  updateChannelInProduct({
      productId,
      channelId,
      isPublished = true,
      isAvailableForPurchase = true,
      visibleInListings = true
```